wgsl/index.bs

```diff
@@ -542,15 +543,16 @@ The following types are <dfn dfn noexport>host-shareable</dfn>:
       <td>Applied to an array type.<br>
          The number of bytes from the start of one element of the array to the
          start of the next element.
-  <tr><td><dfn noexport>`offset`</dfn>
+  <tr><td><dfn noexport>`span`</dfn>
       <td>non-negative i32 literal
       <td>Applied to a member of a structure type.<br>
-         The number of bytes between the start of the structure and the location
-         of this member.
+         In the case where the memory size of a member does not match the size of the member type, a
+         `span` is required. The `span` must be the byte size of memory taken up by the current member and no larger.
```

Comment:
Should we mention somewhere *what* values of the span are expected? This text says it must be "the byte size of memory taken up by the current member" but doesn't explain how to get that.

Reply:
Maybe? I think we can skip it for now, but this maybe an output from the compilers, or a table to show when needed?